### ISSUE TITLE
Quick note of Week 07 homework.

### ISSUE THREAD

**Issue by peperg** [UM-R-for-EnvSci-Registered-Student-2021/General_Discussion#11]
Hello Class!

It was pointed out to me that my reference figures look slightly different than what you get by directly plotting the data i provided. I actually just found a small error in the isntructions of the assignment. My original text had a section indicating that the plots showed the data for the **surface** of the lake (i.e. depth = 0 m), but it seems that that section did not make it to the README. Very sorry about that.

Feel free to add the small filtration step (no more than one line of code for each dataset) to your code if you are still working on it, or if you have time and energy to update your repo. That being said, because this was on me, if you already did it without filtering, i will not consider it wrong.

Sorry about that!

Have a good weekend.

Pepe

**Comment by YananR** [comment]
Thank you! That really save my time to figure out the data!

**Comment by WinsorT** [comment]
I only noticed that the data was different for the phosphorus graph and I tried to figure this out for a couple hours on Wednesday.  The closest I came to replicating it was by grouping the data by sampling dates and I took the minimum of those values.  Kind of interesting to see that the minimum value of each sampling date in each location was similar to the values at depth zero. 

![sampdate_minphos](https://user-images.githubusercontent.com/91145129/139554077-b6278269-b263-42e5-a095-488fcf7bad31.PNG)
![sampdate_minphosC](https://user-images.githubusercontent.com/91145129/139554078-85404274-def3-42df-b009-fe1bd548078b.PNG)
